src/utils/validateApiVersion.js

```diff
@@ -0,0 +1,23 @@
+export const validateApiVersion = version => {
```

Comment:
PP: use export default here

Reply:
@Kemosabert can you change this to keep it consistent with the rest of the modules in the repo?